tests/e2e/ansible/group_vars/all

```diff
@@ -26,7 +26,7 @@ kubeadm_pkgs:
   centos:
     - conntrack
     - socat
-k8s_version: v1.24.0
+k8s_version: v1.28.0
```

Comment:
Is this required by this PR or is it just an addition? If it is required by this PR it'd be nice to mention it in the commit message (like XY requires at least k8s>=Z)

Reply:
According to @wainersm this was used to align with kata ci version. But we can drop for now and introduce this in a different PR. done.